app/config/FeatureSwitch.scala

```diff
@@ -54,4 +54,11 @@ case class FeatureSwitch(value: Option[Configuration]) {
 
     enabled.getOrElse(false)
   }
+
+  def isJacobsNewThing: Boolean = {
+    value.flatMap {
```

Comment:
Without `flatMap` and `getOrElse(false)`, the return type here is `Option[Option[Boolean]]`.

`flatMap` changes it to `Option[Boolean]` and `getOrElse(false)` defaults it to `false` if the Option is None (`value` doesn't exist or there isn't a thing called "is-jacobs-new-thing" in application.conf -> feature-switch).

Reply:
what about using "pattern matching" instead of flatMap? 
value match 
{ 
case Some(x) => x.getConfig("y").....getOrElse(false)
}